scripts/e2e_test.sh

```diff
@@ -34,6 +46,18 @@ source "$BASE_PATH"/scripts/versions.sh
 
 BASEDIR=${BASEDIR:-"$HOME/.teleporter-deps"}
 
+if [ -n "$reuse_network_dir" ]; then
+    if [ -d "$reuse_network_dir" ] && [ "$(ls -A "$reuse_network_dir")" ]; then
```

Comment:
```suggestion
    if [ -d "$reuse_network_dir" ] && [ "$(ls -a "$reuse_network_dir")" ]; then
```

`ls -A` will always return something because it includes `.` and `..`
`ls -a` will omit these